File: app/src/at-phutran/java/asiantech/internship/summer/fragment/WelcomeActivity.java
package asiantech.internship.summer.fragment;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

import asiantech.internship.summer.R;

public class WelcomeActivity extends AppCompatActivity {
    private static final String EMAIL_VALUE = "textEmail";
    private static final String PASSWORD_VALUE = "textPassword";
    private TextView mTvEmail;
    private TextView mTvPassword;
    private String mValueEmail;
    private String mValuePassword;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_welcome);
        mTvEmail = findViewById(R.id.tvEmail);
        mTvPassword = findViewById(R.id.tvPassword);
        Intent intent = getIntent();
        mValueEmail = intent.getStringExtra(EMAIL_VALUE);
        mValuePassword = intent.getStringExtra(PASSWORD_VALUE);
        mTvEmail.setText(mValueEmail);
        mTvPassword.setText(mValuePassword);
    }
}

File: app/src/at-phutran/java/asiantech/internship/summer/kotlin/fragment/WelcomeActivity.kt
package asiantech.internship.summer.kotlin.fragment

import android.annotation.SuppressLint
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import asiantech.internship.summer.R
import kotlinx.android.synthetic.`at-phutran`.activity_welcome.*

@SuppressLint("Registered")
class WelcomeActivity : AppCompatActivity() {
    private var valueEmail = ""
    private var valuePassword = ""
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_welcome)
        valueEmail = intent.getStringExtra(getString(R.string.valueEmail))
        valuePassword = intent.getStringExtra(getString(R.string.valuePassword))
        tvEmail.text = valueEmail
        tvPassword.text = valuePassword
    }
}
